homeassistant/components/bosch_alarm/alarm_control_panel.py

```diff
@@ -0,0 +1,104 @@
+"""Support for Bosch Alarm Panel."""
+
+from __future__ import annotations
+
+from homeassistant.components.alarm_control_panel import (
+    AlarmControlPanelEntity,
+    AlarmControlPanelEntityFeature,
+    AlarmControlPanelState,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import BoschAlarmConfigEntry, BoschAlarmCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant | None,
+    config_entry: BoschAlarmConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up control panels for each area."""
+    coordinator: BoschAlarmCoordinator = config_entry.runtime_data
+
+    async_add_entities(
+        AreaAlarmControlPanel(
+            coordinator,
+            area_id,
+        )
+        for area_id in coordinator.panel.areas
+    )
+
+
+class AreaAlarmControlPanel(
+    CoordinatorEntity[BoschAlarmCoordinator], AlarmControlPanelEntity
+):
+    """An alarm control panel entity for a bosch alarm panel."""
+
+    _attr_has_entity_name = True
```

Comment:
```suggestion
    _attr_has_entity_name = True
    _attr_supported_features = (
        AlarmControlPanelEntityFeature.ARM_HOME
        | AlarmControlPanelEntityFeature.ARM_AWAY
    )
    _attr_code_arm_required = False
```

move static attributes out of constructor